package.json

```diff
@@ -6,7 +6,8 @@
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
     "lint:py": "flake8 --exit-zero",
-    "makemessages": "node ./extract_frontend_messages.js ./contentcuration/contentcuration/static/js/edit_channel/**{/*,}.{js,vue}"
+    "makemessages": "node ./extract_frontend_messages.js ./contentcuration/contentcuration/static/js/edit_channel/**{/*,}.{js,vue}",
+    "build": "webpack --config webpack.config.js"
   },
```

Comment:
Not needed in this PR, I think, but the next step would be to use webpack devserver!

Reply:
So down. Trying to keep this PR minimalist, but adding ideas for iteration in the attached trello board